code/modules/uplink/uplink_items.dm

```diff
@@ -264,6 +264,14 @@ GLOBAL_LIST_INIT(illegal_tech_blacklist, typecacheof(list(
 	cost = 30
 	purchasable_from = UPLINK_NUKE_OPS
 
+/datum/uplink_item/bundles_TC/machoman
+	name = "The Macho Wrestler Bundle"
+	desc = "For the STRONGEST operatives! Master your wrastling abilities with this bundle. \
```

Comment:
I understand what you're going for but Wrastling just looks wrong. Spell it right or find a different way to spell the funny way to say it. `Wrasslin'` has my vote.